### PR TITLE
feature #2465 - my token screen layout

### PR DESCRIPTION
**Pull request by goranjovic**
addresses #2465  
partially addresses #2492 - balance change indicator

### Summary:

Layout for main Assets screen. This is just a layout of the new screen - specific pieces will be implemented in different PRs (see the checklist in #2464) 

The changes are hidden behind a flag and generally not accessible in a normal build.

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/wallet/assets/views.cljs
Comment (jeluard) [inline src/status_im/ui/screens/wallet/assets/views.cljs:33]: For ease of reading we might want to extract some of those as standalone components?

**Review by jeluard — APPROVED** [review]

**Review by oskarth — APPROVED** [review]
LGTM; happy for you to merge it if you don't think it requires QA as it is under flag, your call. Moving to ready for testing by default

**Comment by yenda** [comment]
I don't understand how it addresses #2456

**Comment by goranjovic** [comment]
@yenda Typo! `s/2456/2465`

**Pull request merged**
